packages/web-service/src/pages/returns/a24/artificial-sett-grid-reference.js

```diff
@@ -5,6 +5,7 @@ import { gridReferenceValidator } from '../../../pages/common/grid-ref-validator
 import { APIRequests } from '../../../services/api-requests.js'
 
 const { ARTIFICIAL_SETT_GRID_REFERENCE } = ReturnsURIs.A24
+const { LICENCE_CONDITIONS } = ReturnsURIs
 
 const artificialSettGridReference = 'artificial-sett-grid-reference'
 export const validator = payload => gridReferenceValidator(payload, artificialSettGridReference)
```

Comment:
Did you reuse the one created for A24?

Reply:
licence conditions page is a common one for all species returns